frontend/pages/_app.tsx

```diff
@@ -24,7 +24,7 @@ export default function App({ Component, pageProps }: AppProps) {
                   <meta name="description" content="Made by Joy Almeida" />
                   <link rel="icon" href="/favicon.ico" />
                 </Head>
-                <PageNavbar />
+                <PageNavbar />  
```

Comment:
Please note that these type of changes do not get commited. For people for review and merge your code have to spend time and make sure this is correct. 

simply put, if you make our job easier the fast things get processed.